common_clang.cpp

```diff
@@ -79,6 +79,12 @@ Copyright (c) Intel Corporation (2009-2017).
 #include <thread>
 #endif
 
+namespace SPIRV {
+// Use textual format for SPIRV.
+extern bool SPIRVUseTextFormat;
```

Comment:
I find SPIRVUseTextFormat in llvm-spirv is guarded by macro _SPIRV_SUPPORT_TEXT_FMT. Is this macro always enabled?

Reply:
I think yes. 
SPIRV-LLVM-Translator/lib/SPIRV/SPIRVWriter.cpp includes SPIRVInstruction.h which includes SPIRVStream.h, we can find _SPIRV_SUPPORT_TEXT_FMT is defined in SPIRVStream.h.

Reply:
LGTM. Maybe we can also use convertSpirv instead of using llvm-spirv internal variable.